### PR TITLE
Corrected class name

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

When this pull request is merged, it will...

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/graphql/reference/wishlist.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4054) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4054) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dobooth** [comment]
Hi @yogeshsuhagiya 
Thank you for the contribution.
It looks like this is changing the case of the anchor name.
The anchor name is invisible to the end user, so the actual name used doesn't matter much.
However, the general coding standards for links and anchors are to keep it lowercase.
Thanks again but I will close this PR.

**Comment by yogeshsuhagiya** [comment]
Hi @dobooth, can you please reopen this PR?

**Comment by dobooth** [comment]
My mistake. I missed the camelcase fix. 
We will take this PR today.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/wishlist.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ Use Wishlist queries to retrieve information about a customer's wish list.
 
 Attribute | Data type | Description
 --- | --- | ---
-`items` | [Wishlistitem](#wishlistitem) | An array of items in the customer's wish list
+`items` | [WishlistItem](#WishlistItem) | An array of items in the customer's wish list
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/wishlist.md]: I approve the data type rename to initial cap Item, but we should not update the anchor link. Please remove at part of your change.

File: guides/v2.3/graphql/reference/wishlist.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ Use Wishlist queries to retrieve information about a customer's wish list.
...
 `items_count` | Int | The number of items in the wish list
 `name` | String | When multiple wish lists are enabled, the name the customer assigns to the wish list
 `sharing_code` | String | An encrypted code that Magento uses to link to the wish list
 `updated_at` | String | The time of the last modification to the wish list
 {:style="table-layout:auto;"}
 
-### Wish list item attributes {#wishlistitem}
+### Wish list item attributes {#WishlistItem}
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/wishlist.md]: This anchor link should not be changed. Please remove this part of your edit.

File: guides/v2.3/graphql/reference/wishlist.md (hunk 1 above)
Comment (yogeshsuhagiya) [reply guides/v2.3/graphql/reference/wishlist.md]: Done

File: guides/v2.3/graphql/reference/wishlist.md (hunk 2 above)
Comment (yogeshsuhagiya) [reply guides/v2.3/graphql/reference/wishlist.md]: Done

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @yogeshsuhagiya!

**Pull request merged**